### ISSUE TITLE
Nice Work ! Could you provide the configuration about the SA-1B dataset?

### ISSUE THREAD

**Issue by xuke225** [czg1225/SlimSAM#5]
Hi, could you please describe in detail how the SA-1B dataset is split into the training and the validation set in this work?

**Comment by czg1225** [comment]
You can find the SA-1B dataset from url{https://ai.meta.com/datasets/segment-anything-downloads/}. The SA-1B dataset is divided into multiple subsets, each subset has 11k images. You can download any of the subsets, and then use 10k images as a training set, 50 images as a validationset, and the rest as a testset.  Our experimental results were completed under the above configuration, but if the amount of training data is further increased, the performance of the model will be further improved.
Thanks！

**Comment by czg1225** [comment]
By the way, the 10k training dataset and 50 val dataset are splited randomly from the 11k subset.

**Comment by xuke225** [comment]
Thank you for your response. As you indicated, I only need to download one of the compressed packages in SA-1B (e.g., sa_000000.tar) to complete the training, validation, and testing procedures for this task. However, will the test results have a large variance (due to different choices)?

**Comment by czg1225** [comment]
We have tried different choices. The test results will be similar (MIoU variance < 0.5%).

**Comment by xuke225** [comment]
Got it, thank you for your patient reply！
